src/Requester.ts

```diff
@@ -25,4 +26,12 @@ export class Requester implements IRequester {
             );
         });
     }
+    private getRequestFn(url: string) {
+        //tslint:disable no-http-string
+        if (url.substr(0, 5) === 'http:') {
```

Comment:
Technically using the url parser would be the ideal way, but `startsWith` will do I think.